### DISCUSSION TITLE
unable to create new file

### DISCUSSION THREAD

**Discussion by hunter1Frank** [liamcain/obsidian-calendar-plugin#268] [General]
![image](https://user-images.githubusercontent.com/109122657/191158340-cd2ea591-ff35-41fb-a715-5ba7680fb5f5.png)
When I click “create”，it‘d like this
no dot showed in daily, & can't create new daily-note cliking the date
![image](https://user-images.githubusercontent.com/109122657/191158601-605ee0cd-a8a5-46b6-a52c-a7637d9266f4.png)

**Comment by liamcain** [comment]
That means that your daily note format isn't supported. If you share a screenshot of your settings, I can help point out what the problem is.
